classes/01class/exercises/c01-e03/henriqueediniz/c01-e03.txt

```diff
@@ -0,0 +1,53 @@
+Public IP:
+
+I access the EC2 console and with help of documentation I made a configuration of my first instance. I create one Image of Amazon Linux AMI with a default VPC and I create a security group called Henrique Security Group to allows SSH with Key pairs that I create before named as myKey.
+
+My terminal:
+
+✦ 👽 henriquediniz ssh -i "myKey.pem" ec2-user@ec2-34-238-83-178.compute-1.amazonaws.com
+Last login: Fri Feb  7 12:07:15 2020 from 155.143.178.112
+
+       __|  __|_  )
+       _|  (     /   Amazon Linux 2 AMI
+      ___|\___|___|
+
+https://aws.amazon.com/amazon-linux-2/
+7 package(s) needed for security, out of 39 available
+Run "sudo yum update" to apply all updates.
+[ec2-user@ip-172-31-85-181 ~]$ 
+
+
+
+Documentation reference: https://docs.aws.amazon.com/efs/latest/ug/gs-step-one-create-ec2-resources.html
+
+
+Private IP: 
+
+To access my other instance, I did not flag Assign IPs automatically as private and used the same configuration as the last instance, however, I have to create an elastic IP associated with my private IP to be accessed from within the VPC using the internal IP .
```

Comment:
Hey Henrique, thats good to see that you're going through the documentation to find how to solve the exercise.
In that case, the second instance can not not have any external IP, so an Elastic IP attached to it is not the way to go.
The way to connect to the second instance is through a `jump server` or `bastion host`. This means that you need to access one server that have a public ip(like the first server) and from that public accessible server, you connect to the second server that only have internal IP. That's the reason for the name `jump server`. You first need to get into the first server to "jump" into the second server.
Review that and update your c01-e03.txt file so we can approve, but i'ts great to see your dedication to this!

Reply:
Hi Kiko, I understand what do you want say and made my exercise again. Can you check the answer?